pkgs/racket-doc/scribblings/guide/define.scrbl

```diff
@@ -194,10 +194,10 @@ equivalent to a @racket[define-values] form with a single @racket[_id].
 
 When the grammar for a syntactic form specifies @racket[_body], then
 the corresponding form can be either a definition or an expression.
-A definition as a @racket[_body] is an @defterm{internal definition}.
+A definition as a @racket[_body] is an @deftech{internal definition}.
```

Comment:
"A definition in", I'd say.

Reply:
Are you treating the metavariable `body` as the whole sequence? IIUC, elsewhere, the metavariable `body` is treated as a single form in the sequence (hence the usage of `@racket[body]s` in various places). So I think "a definition as a @racket[_body]" is correct already.